src/app/mods/item-craftable/itemcraftable.ts

```diff
@@ -0,0 +1,110 @@
+import {Mod} from "../mod";
+
+export class ItemCraftable extends Mod {
+    private itemWindow = null
+    private styleTag: HTMLStyleElement;
+    private statuscheckbox = false
+
+    startMod(): void {
```

Comment:
Tu devrais initialiser ton `styleTag` ici, comme ça :
`this.styleTag = window.document.createElement('style');`
Puis définir son contenu de cette manière :
```
 .checkboxitemcraftable {
   text-align: left;
   padding: 3px 3px 3px 30px;
   background-position: -24px calc(50% - 2px);
   background-size: 18px 18px;
   background-repeat: no-repeat;
   background-origin: content-box;
   background-image: url("./assets/ui/checkbox.png")
}

 .checkboxitemcraftable.on {
   background-image: url("./assets/ui/checkbox_checked.png")
}
```

Et l'ajouter au head comme ça
`this.wGame.document.head.appendChild(this.styleTag);`